static/elements/chromedash-timeline.js

```diff
@@ -0,0 +1,218 @@
+        row.push('A', 'Modernized metrics');
+      } else {
+        row.push(null, null);
+      }
+      rowArray.push(row);
+    }
+
+    table.addRows(rowArray);
+
+    table = google.visualization.data.group(table,
+      [{column: 0, modifier: new Date(date.getFullYear(), date.getMonth()), type: 'date'}],
+      [{
+        column: 1,
+        aggregation: google.visualization.data.avg,
+        type: 'number',
+        label: 'Monthly Average',
+      }],
+      [{column: 2, type: 'string'}]
+    );
+
+    const formatter = new google.visualization.NumberFormat({fractionDigits: 6});
+    formatter.format(table, 1); // Apply formatter to percentage column.
+
+    let view = table;
+
+    if (!showAllHistoricalData) {
+      const startYear = (new Date()).getFullYear() - 2; // Show only 2 years of data by default.
+      view = new google.visualization.DataView(table);
+      view.setRows(view.getFilteredRows([{column: 0, minValue: new Date(startYear, 0, 1)}]));
+    }
+
+    const chart = new google.visualization.LineChart(this.$.chart);
+    chart.draw(view, {
+      // title: this.title,
+      // subtitle: 'all channels and platforms',
+      backgroundColor: 'white',
+      legend: {position: 'none'},
+      curveType: 'function',
+      vAxis: {
+        title: '% page loads',
+        // maxValue: 100,
+        minValue: 0,
+      },
+      hAxis: {
+        title: 'Date',
+        format: 'M/yy',
+      },
+      width: '100%',
+      height: '100%',
+      pointSize: 4,
+      series: {0: {color: '#4580c0'}},
+      trendlines: {
+        0: {
+          type: 'linear',
+          opacity: 0.5,
+          color: '#bdd6ed',
+          pointsVisible: false,
+        },
+      },
+    });
+  }
+
+  _updateTimeline() {
+    if (this.selectedBucketId === 1) {
+      return;
+    }
+
+    const prefix = this.useRemoteData ? 'https://www.chromestatus.com' : '';
+
+    const url = prefix + '/data/timeline/' + this.type + this.view +
+              '?bucket_id=' + this.selectedBucketId;
+
+    this._renderHTTPArchiveData();
+
+    fetch(url).then((res) => res.json()).then((response) => {
+      this.drawVisualization(response, this.selectedBucketId, this.showAllHistoricalData);
+    });
+
+    if (history.pushState) {
+      const url = '/metrics/' + this.type + '/timeline/' + this.view + '/' +
+                this.selectedBucketId;
+      history.pushState({id: this.selectedBucketId}, '', url);
+    }
+  }
+
+  _renderHTTPArchiveData() {
+    if (!this.props.length) {
+      return;
+    }
+
+    const feature = this.props.find((el) => el[0] === parseInt(this.selectedBucketId));
+    if (feature) {
+      const featureName = feature[1];
+      const REPORT_ID = '1M8kXOqPkwYNKjJhtag_nvDNJCpvmw_ri';
+      const dsEmbedUrl = `https://datastudio.google.com/embed/reporting/${REPORT_ID}/page/tc5b?config=%7B"df3":"include%25EE%2580%25800%25EE%2580%2580IN%25EE%2580%2580${featureName}"%7D`;
+      this.$.httparchivedata.src = dsEmbedUrl;
```

Comment:
I think `this.$` was a Polymer API to access an element by id. But I don't think it exists on lit-element. https://github.com/Polymer/lit-element/issues/57

@justinfagnani is that correct?

Reply:
I noticed there are a number of elements using `this.$` so it might be worth doing a pass to catch all of those.

Reply:
Justin suggested caching a reference to the elements using the `firstUpdated` callback.

Reply:
Sorry for the noise. This file is WIP.
I just removed all WIP files. This PR is only the features page.

Reply:
oops! my mistake :)